eventhandling/alertEvent.go

```diff
@@ -102,7 +118,7 @@ func ProcessAndForwardAlertEvent(rw http.ResponseWriter, requestBody []byte, log
 }
 
 // createAndSendCE create a new problem.triggered event and send it to Keptn
-func createAndSendCE(problemData keptncommons.ProblemEventData, shkeptncontext string) error {
+func createAndSendCE(problemData remediationTriggeredEventData, shkeptncontext string) error {
```

Comment:
Maybe this type should be a pointer, or the variable `newEventData` or the call to the function shouldn't be one

Reply:
I just removed it from the declaration